crates/doc/src/ptr.rs

```diff
@@ -251,6 +260,7 @@ impl Pointer {
                     Token::Index(ind) => fields.insert_property(&ind.to_string(), alloc),
                     Token::Property(property) => fields.insert_property(property, alloc),
                     Token::NextIndex => fields.insert_property("-", alloc),
```

Comment:
here too: return None. The current `insert_property` handling is only required because we currently parse "-" into NextIndex.